### ISSUE TITLE
Is this still working?

### ISSUE THREAD

**Issue by bjozzi** [HSLdevcom/localroute#15]
I was trying to run this but get stuck on step 2: Preprocess schedule data.
node lr.js --date 2013-12-02 --in-gtfs helsinki/gtfs.zip --out-tempt helsinki/readable.txt --out-gtfs-geom helsinki/geom.txt

After running this line with my gftz file and of course changing the paths to my gtfz file, just nothing happens.  Nothing gets created, no errors, just nothing.

I tried running this on Windows and Ubuntu.

**Comment by diegocerdan** [comment]
@bjozzi Did you make it work? Looks like the only solution available for routing offline with Javascript.

**Comment by bjozzi** [comment]
No, never got it working. Looks promising but guess it is not being maintained in any way.

**Comment by diegocerdan** [comment]
What a pity. There are plenty of use cases where it would be great to offer and offline experience through a modern website. Sure you can't store all data and images from big areas but still useful for isolated little maps.
